src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigDrawer.tsx

```diff
@@ -125,10 +116,9 @@ export const AutomatedAnalysisConfigDrawer: React.FC<AutomatedAnalysisConfigDraw
     drawerRef.current && drawerRef.current.focus();
   }, [drawerRef]);
 
-  const onOptionSelect = React.useCallback(() => {
-    setIsDropdownOpen(false);
+  const onCogSelect = React.useCallback(() => {
     setIsExpanded(!isExpanded);
```

Comment:
Should use factory function notation :D Just a React practice thing.

Reply:
I don't think I can since this is `onClick` on a button where the first parameter is the event.